### DISCUSSION TITLE
No output while running the TB-Profiler in docker

### DISCUSSION THREAD

**Discussion by Meeenakshipari** [jodyphelan/TBProfiler#311] [Q&A]
I am running tb-profiler through docker , with this command .. 
command :

 docker run -v $PWD:$PWD quay.io/biocontainers/tb-profiler:4.3.0--pypyh5e36f6f_0 tb-profiler profile -1 $PWD/file1 -2 $PWD/file2 -p test

Sample output where it said the profiler has completed successfully 

Writing outputs
---------------
Writing json file: ./results/test.results.json
Writing text file: ./results/test.results.txt

Running command:
set -u pipefail; rm ./cb84b5bf-e119-4de5-9a7e-be97479a453b*

Profiling finished sucessfully!

But no file is created 
This is what it is showing but not file is generated in the current or any working directory , 

Can anyone help in understanding where I am going wrong I tried checking online for any issues with the docker command , unable to come to a conclusion ,Since I am very new to docker and tb-profiler any suggestions would be helpful 

Thanks in advance

**Comment by jodyphelan** [comment]
Hi @Meeenakshipari ,

Did you manage to figure this one out or is it still an issue?
